pom.xml

```diff
@@ -29,6 +29,8 @@
     <properties>
         <java.version>21</java.version>
         <mapstruct.version>1.6.3</mapstruct.version>
+        <junit.version>4.13.2</junit.version>
```

Comment:
To jest błąd -> dodając Spring-Boot-Tests ona ma w sobie JUNITa w wersji 5 aktualnie masz JUINIT w wersji 4 oraz JUNIT w wersji 5 dodany do projektu co może powodować problemy w jego działaniu.